framework/build.mk

```diff
@@ -29,7 +29,7 @@ libmesh_CC       ?= $(shell METHOD=$(METHOD) $(libmesh_config) --cc)
 libmesh_F77      ?= $(shell METHOD=$(METHOD) $(libmesh_config) --fc)
 libmesh_F90      ?= $(shell METHOD=$(METHOD) $(libmesh_config) --fc)
 libmesh_INCLUDE  := $(shell METHOD=$(METHOD) $(libmesh_config) --include)
-libmesh_CPPFLAGS := $(shell METHOD=$(METHOD) $(libmesh_config) --cppflags)
+libmesh_CPPFLAGS := $(shell METHOD=$(METHOD) $(libmesh_config) --cppflags) $(ADDITIONAL_CPPFLAGS)
```

Comment:
That's what I did before, but it appears in 6 or seven spots :-/. But I can change it.